lib/manageiq/reporting/formatter/c3.rb

```diff
@@ -1,183 +1,190 @@
-module ReportFormatter
-  class C3Formatter < Ruport::Formatter
-    include ActionView::Helpers::UrlHelper
-    include ChartCommon
-    include MiqReport::Formatting
-    renders :c3, :for => ReportRenderer
-
-    # series handling methods
-    def series_class
-      C3Series
-    end
-
-    CONVERT_TYPES = {
-      "ColumnThreed"         => "Column",
-      "ParallelThreedColumn" => "Column",
-      "StackedThreedColumn"  => "StackedColumn",
-      "PieThreed"            => "Pie",
-      "AreaThreed"           => "Area",
-      "StackedAreaThreed"    => "StackedArea"
-    }
-    def add_series(label, data)
-      @counter ||= 0
-      @counter += 1
-      series_id = @counter.to_s
-      limit = pie_type? ? LEGEND_LENGTH : LABEL_LENGTH
-
-      if chart_is_2d?
-        mri.chart[:data][:columns] << [series_id, *data.map { |a| a[:value] }]
-        mri.chart[:data][:names][series_id] = slice_legend(_(label), limit)
-        mri.chart[:miq][:name_table][series_id] = label
-      else
-        data.each_with_index do |a, index|
-          id = index.to_s
-          mri.chart[:data][:columns].push([id, a[:value]])
-          mri.chart[:data][:names][id] = slice_legend(_(a[:tooltip]), limit)
-          mri.chart[:miq][:name_table][id] = a[:tooltip]
+require_dependency 'manageiq/reporting/formatter/c3_series'
+
+module ManageIQ
+  module Reporting
+    module Formatter
+      class C3 < Ruport::Formatter
+        include ActionView::Helpers::UrlHelper
+        include ChartCommon
+        include MiqReport::Formatting
+        renders :c3, :for => ReportRenderer
+
+        # series handling methods
+        def series_class
+          # ManageIQ::Reporting::Formatter::C3Series
```

Comment:
Any reason not to use the fully qualified name rather than just commenting it?

Reply:
DOH.  I'll fix... Good eye.  I did this while testing the deprecation stuff. 👁